File: src/LightNap.Core/Users/Dto/Response/AdminUserDto.cs
namespace LightNap.Core.Users.Dto.Response
{
    /// <summary>
    /// Represents a user DTO with public information.
    /// </summary>
    public class AdminUserDto : PrivilegedUserDto
    {
        /// <summary>
        /// The last modified date of the user.
        /// </summary>
        public long LastModifiedDate { get; set; }

        /// <summary>
        /// When the user's lockout ends, if applicable.
        /// </summary>
        public long? LockoutEnd { get; set; }
    }
}

File: src/lightnap-ng/src/app/core/backend-api/dtos/users/response/admin-user-dto.ts
import { PrivilegedUserDto } from "./privileged-user-dto";

/**
 * Interface representing a user with full details for an administrative context.
 */
export interface AdminUserDto extends PrivilegedUserDto {
    /**
     * The timestamp when the user was last modified.
     * @type {number}
     */
    lastModifiedDate: number;

    /**
     * The timestamp when the user lockout ends. If the user is not locked out, this value is undefined.
     * @type {number}
     */
    lockoutEnd: number;
}
